packages/ngtools/webpack/src/compiler_host.ts

```diff
@@ -354,13 +357,46 @@ export class WebpackCompilerHost implements ts.CompilerHost {
   trace(message: string) {
     console.log(message);
   }
-}
 
+  resolveModuleNames(
+    moduleNames: string[],
+    containingFile: string,
+  ): (ts.ResolvedModule | undefined)[] {
+    return moduleNames.map(moduleName => {
+      const { resolvedModule } = ts.resolveModuleName(
```

Comment:
Thinking about this more.  We are essentially resolving modules using two completely different code paths in a single compilation.   Transpilation is using the TS built-in resolution and bundling is using webpack's.  This seems particularly error prone and fragile.  While they ideally should resolve to the same place, there is no guarantee; especially when developers are including this plugin in a custom webpack configuration (a use case which should be fully supported).

Should this be deferring to webpack's module resolution for consistency?

Reply:
Maybe indeed we should change this. Can we have a separate chat and maybe address this in a separate PR. Pinged you already on slack.